compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -85,16 +88,24 @@ llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
       reorderStr = "transpose";
     } else if (options.reorderStrategy == ReorderWorkgroupsStrategy::Swizzle) {
       reorderStr = "swizzle";
+    } else if (options.reorderStrategy ==
+               ReorderWorkgroupsStrategy::ChipletGroup) {
+      reorderStr = "chilpletgroup";
     } else {
       assert(options.reorderStrategy == ReorderWorkgroupsStrategy::None &&
              "Unhandled reorder option");
       reorderStr = "none";
     }
   }
 
+  StringRef tilesizeStr = "<not set>";
+  if (options.reorderWgLogTileSize)
+    tilesizeStr = StringRef(
+        std::to_string((unsigned long)options.reorderWgLogTileSize.value()));
   return os << "{" << "enableReduceSharedMemoryBankConflicts = "
             << options.enableReduceSharedMemoryBankConflicts
             << ", reorderWorkgroupsStrategy = " << reorderStr
+            << ", reorderWorkgroupsTileSize (log2) = " << tilesizeStr
```

Comment:
Use the actual field names from `LLVMGPUPipelineOptions` (`reorderWgLogTileSize`)